src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -117,13 +121,8 @@ static String getParentPath( final String path ) {
 
 	@Override
 	public DefaultValueFinder getDefaultValueFinder() {
-		return new DefaultValueFinder() {
-			@Override
-			public Serializable getDefaultValue( final IdentifyingAttributes comp, final String attributesKey ) {
-				// TODO Auto-generated method stub
-				return null;
-			}
-		};
+		// TODO DefaultValueFinder is just a stub.
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=9&issues=AWQTHhXyYwZU9vvrXIVC&open=AWQTHhXyYwZU9vvrXIVC)

[ISSUE_KEY]: AWQTHhXyYwZU9vvrXIVC